SU-SWS/decanter-react
# ADAPT-000: Add Identity Bar, Logo, Global Footer, SR only Text, Button

# READY

# Summary
- Add Logo, Identity Bar, Global Footer, Button, SrOnlyText components

# Needed By (Date)
- When does this need to be merged by?

# Urgency
- How critical is this PR?

# Steps to Test

1. Pull down to local, nvm use, npm run dev
1. Look at Storybook local site and check that the new components are there
1. Look at code

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
